lib/src/neuronGroup.cc

```diff
@@ -88,6 +88,74 @@ void NeuronGroup::calcSizes(unsigned int blockSize,  unsigned int &idStart, unsi
     m_PaddedIDRange.second = paddedIDStart;
 }
 
+void NeuronGroup::mergeIncomingPSM()
+{
+    // Create a copy of this neuron groups incoming synapse populations
+    std::vector<SynapseGroup*> inSyn = getInSyn();
+
+    // Loop through un-merged incoming synapse populations
+    for(unsigned int i = 0; !inSyn.empty(); i++) {
+        // Remove last element from vector
+        SynapseGroup *a = inSyn.back();
+        inSyn.pop_back();
```

Comment:
isn't pop_back returning the last element of the vector, i.e. allowing
```SynapseGroup *a = inSyn.pop_back();```
?

Reply:
Sadly it doesn't - it must have for **some** programming language though because in my head it should too